app/translations.js

```diff
@@ -2530,6 +2540,16 @@ const translations = {
     navigate: 'Navigoi',
     'navigation-description': 'Löydä perille ohjatusti',
     'navigation-header': 'Matkan seuranta',
+    'navigation-intro': 'TODO_navigation-intro_EN',
```

Comment:
Will this become used in design plans?

Reply:
True. Must be an oversight leftover. I'll remove this key.